app/assets/javascripts/backbone/views/loan_questions_view.coffee

```diff
@@ -149,3 +155,20 @@ class MS.Views.LoanQuestionsView extends Backbone.View
         "<div class='loan-type-required'>#{loan_type}</div>"
       .join(' ') +
       "</div>"
+
+  permittedActionsHTML: (node) ->
+    if node.can_edit == true
```

Comment:
I don't think you need `== true` here. `== true` is very rarely needed. Only if you are checking that something is exactly `true` versus some other "truthy" value. And in that case, you'd want `===` I think. But as I say, that is rare.